### DISCUSSION TITLE
Does espresense support multiple devices for a single room?

### DISCUSSION THREAD

**Discussion by rosstroha** [ESPresense/ESPresense#600] [Support]
My basement is oddly shaped, so there's a need for multiple ESPresense devices to track occupancy. Is it possible to have more than one ESPresense device publish to same topic, but not collide in Home Assistant?

When I set the room to just `basement` on two different devices, Home Assistant shows the devices constantly connecting and disconnecting.

**Comment by Buckeyes1995** [comment]
Couldn't you just change the 'room' setting to be different?  basement_1 and basement_2 for example?  Your automations could just take both into account when determining if you're in the basement.

**Reply by rosstroha** [reply]
I suppose I could, but I am trying to avoid the added complexity of essentially having two rooms being considered one

**Reply by DTTerastar** [reply]
Room names are required to be unique because we use it for our mqtt client id.
